ui/lib/core/addon/templates/components/confirm.hbs

```diff
@@ -0,0 +1,17 @@
+{{! template-lint-disable no-inline-styles}}
```

Comment:
is this still necessary with the `htmlSafe` declaration?

Reply:
it's not, nice catch!